package.json

```diff
@@ -14,6 +14,7 @@
     "debug": "electron-vite --remote-debugging-port=9223",
     "build": "electron-vite build",
     "compile": "electron-builder --config .electron-builder.config.js",
+    "release": "electron-builder --config .electron-builder.config.js",
```

Comment:
what's the reason for this script compared to the existing compile script?

Reply:
https://www.electron.build/configuration/publish#how-to-publish 

> Add to scripts in the development package.json:
>
> "release": "electron-builder"
>
>and if you run yarn release, a release will be drafted (if doesn’t already exist) and artifacts published.

Reply:
If I understand the docs right it's either this or calling `electron-builder --config .electron-builder.config.js --publish onTag` directly, also it might be just enough to run `npm compile` because it autodetects it's running in CI and defaults to `--publish onTag`